src/tools/build-manifest/src/main.rs

```diff
@@ -167,6 +168,9 @@ struct Builder {
     rust_version: String,
     cargo_version: String,
     rls_version: String,
+    rust_git_commit_hash: Option<String>,
+    cargo_git_commit_hash: Option<String>,
```

Comment:
It looks like these are never initialized?

Reply:
Fixed.